core-services/release-controller/deploy-ocp-controller-ppc64le.yaml

```diff
@@ -0,0 +1,124 @@
+      insecureEdgeTerminationPolicy: Redirect
+    to:
+      kind: Service
+      name: release-controller-ocp-ppc64le
+- kind: Deployment
+  apiVersion: apps/v1
+  metadata:
+    name: release-controller-ocp-ppc64le
+    namespace: ci
+    annotations:
+      image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"release-controller:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"controller\")].image"}]'
+  spec:
+    selector:
+      matchLabels:
+        app: release-controller-ocp-ppc64le
+    template:
+      metadata:
+        labels:
+          app: release-controller-ocp-ppc64le
+      spec:
+        serviceAccountName: release-controller-ocp
+        volumes:
+        - configMap:
+            defaultMode: 420
+            name: config
+          name: config
+        - configMap:
+            defaultMode: 420
+            name: job-config-misc
+          name: job-config-misc
+        - configMap:
+            defaultMode: 420
+            name: job-config-master
+          name: job-config-master
+        - configMap:
+            defaultMode: 420
+            name: job-config-3.x
+          name: job-config-3x
+        - configMap:
+            defaultMode: 420
+            name: job-config-4.0
+          name: job-config-40
+        - configMap:
+            defaultMode: 420
+            name: job-config-4.1
+          name: job-config-41
+        - configMap:
+            defaultMode: 420
+            name: job-config-4.2
+          name: job-config-42
+        - configMap:
+            defaultMode: 420
+            name: job-config-4.3
+          name: job-config-43
+        - configMap:
+            defaultMode: 420
+            name: job-config-4.4
+          name: job-config-44
+        containers:
+        - name: controller
+          image: release-controller:latest
+          volumeMounts:
+          - mountPath: /etc/config
+            name: config
+            readOnly: true
+          - mountPath: /etc/job-config/misc
+            name: job-config-misc
+            readOnly: true
+          - mountPath: /etc/job-config/master
+            name: job-config-master
+            readOnly: true
+          - mountPath: /etc/job-config/3.x
+            name: job-config-3x
+            readOnly: true
+          - mountPath: /etc/job-config/4.0
+            name: job-config-40
+            readOnly: true
+          - mountPath: /etc/job-config/4.1
+            name: job-config-41
+            readOnly: true
+          - mountPath: /etc/job-config/4.2
+            name: job-config-42
+            readOnly: true
+          - mountPath: /etc/job-config/4.3
+            name: job-config-43
+            readOnly: true
+          - name: job-config-44
+            mountPath: /etc/job-config/4.4
+            readOnly: true
+          command:
+          - /usr/bin/release-controller
+          - --to=release
+          - --release-namespace=ocp-ppc64le
+          - --release-namespace=origin
```

Comment:
I dont think you need to monitor the image streams in the **origin** namespace

Reply:
Removing in next push, but it was actually part of the original: https://github.com/openshift/release/blob/24b8097dfe501d458eb758e9106cd5450acf2b7e/core-services/release-controller/deploy-ocp-controller.yaml#L143-L144  :)

Reply:
Yes, the **ocp** release-controller is configured that way because it creates releases and then mirrors into **origin**.  I do not think you'll need, or want, to do that with these new controllers.